tests/infra/network.py

```diff
@@ -1047,7 +1047,7 @@ def wait_for_all_nodes_to_commit(self, primary=None, tx_id=None, timeout=10):
                 with node.client() as c:
                     logs[node.node_id] = []
                     resp = c.get(
-                        f"/node/local_tx?transaction_id={tx_id}",
+                        f"/node/tx?transaction_id={tx_id}",
```

Comment:
Once upon a time, `local_tx` was never forwarded, and that was necessary for some tests to pass. Looks like that's no longer the case though, so if the CI is happy let's purge it.

Reply:
Yes, I think it ceased being useful when "set_execute_outside_consensus" was removed.